packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.constants.js

```diff
@@ -0,0 +1,22 @@
+export const BACKUP_OFFER_LEGACY = 'legacy';
+export const BACKUP_OFFER_PREMIUM = 'premium';
+export const BACKUP_OFFER_ADVANCED = 'advanced';
+export const BACKUP_OFFER_CLASSIC = 'classic';
+
+export const BACKUP_STATE_DISABLED = 'disabled';
+export const CATALOG_INFO = {
+  ADDON_FAMILY_NAME: 'backup',
+  ADDON_PLAN_CODE: 'pcc-option-backup-managed',
+  ADDON_BACKUP_JOB_OFFER_FAMILY_NAME: 'backupjob-offer',
+};
+export const VEEAM_TARIFF_DETAILS_URL = 'https://www.ovhcloud.com/';
```

Comment:
No dependencies on `ovhSubsidiary` or `region` ?

Reply:
We got the urls just recently for each subsidiary and the same has been updated. Thanks